src/utils/skipLogic.js

```diff
@@ -40,8 +40,12 @@ module.exports = function () {
     if (formField.skipLogic.checks.length > 0) {
       for (var i = 0; i < formField.skipLogic.checks.length; i++) {
         var check = formField.skipLogic.checks[i]
-        scope.$watch(check.variable, function (value, oldValue) {
-          skipLogicOperandCheck(scope, value, check)
+        scope.$watch(check.variable.startsWith('form.') ? check.variable : 'form.' + check.variable + '.$modelValue', function (value, oldValue) {
```

Comment:
just incase the parameter is passed in by name, this should handle it

Reply:
Wouldn't we also have to add watchers for all variables in a group. For groups to work correctly we need to watch each field that is part of the check. I don't think that is covered here.